operator-catalog-initialization-test.yml

```diff
@@ -5,7 +5,7 @@
   gather_facts: false
 
   vars:
-    operator_work_dir: "/home/jenkins/agent/test-operator"
+    operator_work_dir: "/home/jenkins/test-operator"
     work_dir: "{{ lookup('env', 'WORKSPACE') }}"
     offline_cataloger_bin_path: "/usr/local/bin/offline-cataloger"
     production_quay_namespace: "redhat-operators"
```

Comment:
I would remove this change from the patch.
I asked Scott to unify names in initial pull request. because on some places we use `/home/jenkins/agent/test-operator` and on other `home/jenkins/test-operator`. It should not case any isseu in downstream.